pymeasure/instruments/anritsu/anritsuMS2090A.py

```diff
@@ -0,0 +1,516 @@
+        "FETCh:GPS:FULL?",
+        '''
+        Returns the timestamp, latitude, longitude, altitude, and satellite count of the device.
+        '''
+    )
+
+    gps_all = Instrument.measurement(
+        "FETCh:GPS:ALL?",
+        '''
+        Returns the fix timestamp, latitude, longitude, altitude and information on the sat used.
+        '''
+    )
+
+    gps = Instrument.measurement(
+        ":FETCh:GPS?",
+        '''
+        Returns the timestamp, latitude, and longitude of the device.
+        '''
+    )
+
+    gps_last = Instrument.measurement(
+        ":FETCh:GPS:LAST?",
+        '''
+        Returns the timestamp, latitude, longitude, and altitude of the last fixed GPS result.
+        '''
+    )
+
+    external_current = Instrument.measurement(
+        "BIAS:EXT:CURR?",
+        '''
+        This command queries the actual bias current in A
+        '''
+    )
+
+    ####################################
+    # Spectrum Parameters - Wavelength #
+    ####################################
+
+    frequency_center = Instrument.control(
+        "FREQuency:CENTer?", "FREQuency:CENTer %s",
+        "Sets the center frequency in Hz",
+        validator=double_validation_value_and_freq,
+        values=[[-99999999995, 299999999995], UNITSFREQ]
+    )
+
+    frequency_offset = Instrument.control(
+        "FREQuency:OFFSet?", "FREQuency:OFFSet %s",
+        "Sets the frequency offset in Hz",
+        validator=double_validation_value_and_freq,
+        values=[[-10000000000, 10000000000], UNITSFREQ],
+    )
+
+    frequency_span = Instrument.control(
+        "FREQuency:SPAN?", "FREQuency:SPAN %s",
+        "Sets the frequency span in Hz",
+        validator=double_validation_value_and_freq,
+        values=[[10, 400000000000], UNITSFREQ],
+    )
+
+    frequency_span_full = Instrument.setting(
+        "FREQuency:SPAN:FULL",
+        "Sets the frequency span to full span"
+    )
+
+    frequency_span_last = Instrument.setting(
+        "FREQuency:SPAN:LAST",
+        "Sets the frequency span to the previous span value."
+    )
+
+    frequency_start = Instrument.control(
+        "FREQuency:STARt?", "FREQuency:STARt %s",
+        "Sets the start frequency in Hz",
+        validator=double_validation_value_and_freq,
+        values=[[-100000000000, 299999999990], UNITSFREQ],
+    )
+
+    frequency_step = Instrument.control(
+        ":FREQuency:STEP?", ":FREQuency:STEP %s",
+        "Set or query the step size to gradually increase or decrease frequency values in Hz",
+        validator=double_validation_value_and_freq,
+        values=[[0.1, 1000000000], UNITSFREQ],
+    )
+
+    frequency_stop = Instrument.control(
+        "FREQuency:STOP?", "FREQuency:STOP %s",
+        "Sets the start frequency in Hz",
+        validator=double_validation_value_and_freq,
+        values=[[-99999999990, 300000000000], UNITSFREQ],
+    )
+
+    abort = Instrument.setting(
+        "ABOR",
+        '''
+        Abort measure
+        '''
+    )

```

Comment:
An "action", like abort, should be a method, not a property, as you do not set anything, see [documentation](https://pymeasure.readthedocs.io/en/latest/dev/adding_instruments.html#your-instrument-s-user-interface).